src/main/java/Storage.java

```diff
@@ -0,0 +1,112 @@
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Storage {
+
+    String path;
+
+    Storage(String filePath) {
+        this.path = filePath;

```

Comment:
In a few cases you pass the path name as string, and here you are using a variable. These appear inconsistent.